INSTALL

```diff
@@ -5,7 +5,19 @@ How to install PBS using the configure script.
 
 1. Install the prerequisite packages for building PBS.
 
-  For CentOS systems you should run the following command as root:
+  For CentOS-8 systems you should run the following command as root:
+
+    Configure and enable PowerTools repo for hwloc-devel and libedit-devel packages
```

Comment:
This instruction should start a column before so that the dnf below can be in-lined with the other dnf.